packages/core/content-manager/admin/src/pages/EditView/components/FormLayout.tsx

```diff
@@ -58,7 +70,7 @@ const FormLayout = ({ layout, hasBackground = false }: FormLayoutProps) => {
         return (
           <Box
             key={index}
-            {...(!hasBackground && {
+            {...(hasBackground && {
```

Comment:
again i really wish we found a better name for this, as neither true or false is really clear

Reply:
maybe hasSinglePane? the biggest difference is that we're not splitting the different layout panes, they're all in the same box with a common background

Reply:
ah ok thanks